reports/menubar-app-to-check-who-is-on-vacation.md

```diff
@@ -0,0 +1,108 @@
+# Menubar app to check who is on vacation
+
+The corresponding challenge is [#92](https://github.com/SolidLabResearch/Challenges/issues/92).
+
+## Problem
+<!--
+You can reuse the pitch of the challenge, but check if you need to make changes.
+For example, it might happen that the approved solution does more than what the original pitch requested.
+-->
+
+Manually checking your calendar to quickly see who is on vacation is a hassle. 
+It would be easier to have menubar app that shows in a glance who is on vacation today.
+
+## Approved solution
+<!--
+Provide information about the approved solution:
+names of tools/libraries created, repos, and so on.
+-->
+
+We developed a [menubar app](https://github.com/SolidLabResearch/solid-menubar-app) that
+shows who is when on vacation.
+You find a screencast of the app [here](https://cloud.ilabt.imec.be/index.php/s/HEeGsCS4PiQxaS7)
+and below you find a screenshot.
+
+![img.png](img/solid-menubar-app.png)
+
+## User flow
+
+<!--
+Describe a concrete user flow with the approved solution.
+Complete the following sections:
+-->
+
+### Actors/actresses
+
+- Menubar app
+- User of the application
+
+### Preconditions
+
+- The user has Node.js installed.
+
+### Steps
+
+1. Clone [the repo of the app](https://github.com/SolidLabResearch/solid-menubar-app) via
+   ```shell
+   git clone https://github.com/SolidLabResearch/solid-menubar-app.git
+   ```
+2. Install the dependencies via
+   ```
+   npm i
+   ```
+3. Navigate to the folder `pod-example-data` via
+   ```
+   cd pod-example-data
+   ```
+4. Populate the test pods on the [SolidLab Playground](https://pod.playground.solidlab.be/) via 
+   ```shell
+   node create-example-pods.js
+   ```
+5. Go back to the root of the repo via
+   ```shell
+   cd ..
+   ```
+6. Copy the example config file that works with the test pods via
+   ```shell
+   cp pod-example-data/config.json config.json
+   ```
+7. Start the app via 
+   ```shell
+   npm start
+   ```
+8. The app appears in the menubar. The icon is an umbrella on a beach.
+
+### Postconditions
+
+- When you click on the icon,
+it shows the vacation days.
+
+## Follow-up actions
```

Comment:
Who will do them by when?

Reply:
It's not clear to me who is responsible to make sure that the follow-up actions are acted upon, but if that's just how it is, then that's a different issue and not relevant for this review.

Reply:
Yeah, that's out-of-scope of the report.